neurodamus/core/configuration.py

```diff
@@ -803,31 +796,6 @@ def _randomize_gaba_risetime(config: _SimConfig):
     h.randomize_Gaba_risetime = randomize_risetime
 
 
-@SimConfig.validator
-def _current_dir(config: _SimConfig):
-    curdir = config.run_conf.get("CurrentDir")
-
-    if curdir is None:
-        log_verbose("CurrentDir using simulation config path [default]")
-        curdir = config.simulation_config_dir
-    else:
-        if not os.path.isabs(curdir):
-            if curdir == ".":
-                logging.warning(
-                    "Setting CurrentDir to '.' is discouraged and "
-                    "shall never be used in production jobs."
-                )
-            else:
-                raise ConfigurationError("CurrentDir: Relative paths not allowed")
-            curdir = os.path.abspath(curdir)
-        if not os.path.isdir(curdir):
-            raise ConfigurationError("CurrentDir doesnt exist: " + curdir)
-
-    log_verbose("CurrentDir = %s", curdir)
```

Comment:
Shall we keep this verbose log somewhere in order to know the path to find the simulation data, e.g. coreneuron data dir? Maybe print the log where `simulation_config_dir` is determined.